rollup.config.js

```diff
@@ -1,32 +1,69 @@
-import commonjs from 'rollup-plugin-commonjs';
-import nodeResolve from 'rollup-plugin-node-resolve';
-import nodeBuiltins from 'rollup-plugin-node-builtins';
-import babel from 'rollup-plugin-babel';
-import sourcemaps from 'rollup-plugin-sourcemaps';
-import { uglify } from "rollup-plugin-uglify";
-import banner from 'rollup-plugin-banner';
-import genHeader from './lib/header';
+import commonjs from 'rollup-plugin-commonjs'
```

Comment:
please don't change existing code if you don't have to.
Cody style changes ALWAYS should be done in a seperate pull-request.